commands/certmanager/post.go

```diff
@@ -0,0 +1,135 @@
+package certmanager
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/spf13/cobra"
+)
+
+var createProperties = sdkgo.CertificatePostPropertiesDto{}
+
+func CertPostCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace:  "certmanager",
+		Resource:   "certificates",
+		Verb:       "create",
+		Aliases:    []string{"c"},
+		ShortDesc:  "Add a new Certificate",
+		LongDesc:   "Use this command to add a Certificate.",
+		Example:    "ionosctl certificate-manager create",
+		PreCmdRun:  PreCmdPost,
+		CmdRun:     CmdPost,
+		InitClient: true,
+	})
+
+	cmd.AddStringFlag(CertName, "", "", "Specify name of the certificate (required either this or --certificate-name-path)")
+	cmd.AddStringFlag(Cert, "", "", "Specify the certificate itself (required either this or --certificate-path)")
+	cmd.AddStringFlag(CertChain, "", "", "Specify the certificate chain (required either this or --certificate-chain-path)")
+	cmd.AddStringFlag(PrivateKey, "", "", "Specify the private key (required either this or --private-key-path)")
+
+	cmd.AddStringFlag(CertNamePath, "", "", "Specify name of the certificate from a file (required either this or --certificate-name)")
+	cmd.AddStringFlag(CertPath, "", "", "Specify the certificate itself from a file (required either this or --certificate)")
+	cmd.AddStringFlag(CertChainPath, "", "", "Specify the certificate chain from a file (required either this or --certificate-chain)")
+	cmd.AddStringFlag(PrivateKeyPath, "", "", "Specify the private key from a file (required either this or --private-key)")
+
+	_ = cmd.Command.RegisterFlagCompletionFunc(ArgCols, func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return allCols, cobra.ShellCompDirectiveNoFileComp
+	})
+
+	return cmd
+}
+
+func SetProperties(c *core.CommandConfig, property string) (string, error) {
+	propertyValue, err := c.Command.Command.Flags().GetString(property)
+	if propertyValue == "" {
+		if err != nil {
+			return "", err
+		}
+		propertyPathName := fmt.Sprintf("%s-path", property)
+		propertyPath, err := c.Command.Command.Flags().GetString(propertyPathName)
+		if err != nil {
+			return "", err
+		}
+		propertyBytes, err := os.ReadFile(propertyPath)
+		if err != nil {
+			return "", err
+		}
+		propertyValue = string(propertyBytes)
+		return propertyValue, nil
+	}
+	return propertyValue, nil
+}
+
+func CmdPost(c *core.CommandConfig) error {
+	c.Printer.Verbose("Adding Certificate...")
+	var name, certificate, certificateChain, privateKey string
+
+	name, err := SetProperties(c, CertName)
```

Comment:
`SetProperties` functionality is nice. However, it's a bit unclear. We are lazy devs and want to know what a function does just by its signature. Also, the fact that it looks into `{certName}-path` too is not "advertised".

1. Rename to reflect above. What do you think about `GetPropertyWithFallback`?
2. Hardcoding the fact that `{certName}-path` is a fallback flag isn't good. Take the path params as an argument too: `GetPropertyWithFallback(c, certName, certNamePath)`